Tests/Metadata/Metadata.csproj

```diff
@@ -12,39 +12,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>$(DefineConstants);CCI</DefineConstants>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'FxCop|AnyCPU' ">
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'v4Debug|AnyCPU' ">
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'v4Release|AnyCPU' ">
-  </PropertyGroup>
-  <ItemGroup>
-    <Compile Include="AssemblyList.cs" />
-    <Compile Include="MetadataTablesExtensions.cs" />
-    <Compile Include="NamespaceTree_Reflection.cs" />
-    <Compile Include="NamespaceTree.cs" />
-    <Compile Include="Program.cs" />
-    <Compile Include="ReflectionExtensions.cs" />
-    <Compile Include="RubyModules.cs" />
-    <Compile Include="Sequence.cs" />
-    <Compile Include="TestCases.cs" />
-    <Compile Include="TypeNestings.cs" />
-    <Compile Include="UnitTests.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\Src\Microsoft.Dynamic\Microsoft.Dynamic.csproj">
-      <Project>{EB66B766-6354-4208-A3D4-AACBDCB5C3B3}</Project>
-      <Name>Microsoft.Dynamic</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\Src\Microsoft.Scripting.Metadata\Microsoft.Scripting.Metadata.csproj">
-      <Project>{ACDD9B9E-8FE6-439C-9521-1CCBA47F6143}</Project>
-      <Name>Microsoft.Scripting.Metadata</Name>
-    </ProjectReference>
+    <ProjectReference Include="..\..\Src\Microsoft.Scripting.Metadata\Microsoft.Scripting.Metadata.csproj" />
+    <ProjectReference Include="..\..\Src\Microsoft.Scripting\Microsoft.Scripting.csproj" />
```

Comment:
In the AssemblyInfo files for Microsoft.Scripting and Microsoft.Dynamic there are some `InternalsVisibleTo("TestInternalDLR, ...")` which may be what's causing the issues. You can comment those out.